package.json

```diff
@@ -30,7 +30,7 @@
     "osd:ciGroup3": "echo apps/vis-augmenter/*.js",
     "osd:ciGroup4": "echo dashboard_sample_data_with_datasource_spec.js,dashboard_sanity_test_spec.js,apps/telemetry/*.js",
     "osd:ciGroup5": "echo datasource-management-plugin/*.js",
-    "osd:ciGroup6": "echo apps/data_explorer/aaa_before.spec.js,apps/data_explorer/data_source_selector.spec.js,apps/data_explorer/date_nanos_mixed.spec.js,apps/data_explorer/date_nanos.spec.js,apps/data_explorer/discover_histogram.spec.js,apps/data_explorer/discover.spec.js,apps/data_explorer/zzz_after.spec.js",
+    "osd:ciGroup6": "echo apps/data_explorer/aaa_before.spec.js,apps/data_explorer/data_source_selector.spec.js,apps/data_explorer/date_nanos_mixed.spec.js,apps/data_explorer/date_nanos.spec.js,apps/data_explorer/discover_histogram.spec.js,apps/data_explorer/discover.spec.js,apps/data_explorer/zzz_after.spec.js,apps/data_explorer/discover_advanced_settings.spec.js,apps/data_explorer/discover_table.spec.js",
```

Comment:
should we put these tests before the `zzz_after.spec.js`?

Reply:
I think it makes more sense to put it before zzz_after.spec.js since we are doing cleanup as part of this suite which might affect the added test suites. I have updated the PR with this change.